### ISSUE TITLE
About custom datasets

### ISSUE THREAD

**Issue by Maojianzeng** [open-mmlab/mmfewshot#78]
I want to use mmfewshot to perform fewshot detection in a custom data set (coco format), but the guidance documents related to the use of custom data sets are too vague. I really can't figure out how to divide the data set, put it to several folders, and I don't know which file to change the settings in. I hope that the guidance document in this regard can be written in more detail, thank you.

**Comment by isJunCheng** [comment]
exactly! According to my own understanding, I divide the custom data set into base and novel samples, but the loss will not decrease after training. I don't know why!
![image](https://user-images.githubusercontent.com/44466856/168405552-c29a5446-7c1c-4c77-af68-4c18d2559c49.png)

**Comment by isJunCheng** [comment]
For mmfewshot, it's strange why there is no discussion

**Comment by Caltech-Z** [comment]
> 确切地！ 按照我自己的理解，我把custom data set分成base和novel samples，但是训练后loss并没有减少。 我不知道为什么！ ![image](https://user-images.githubusercontent.com/44466856/168405552-c29a5446-7c1c-4c77-af68-4c18d2559c49.png)

How did you customize the dataset? How are the folders placed? What are the files in the folder? What do they correspond to? Thank you very much.
